doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -899,11 +905,11 @@ Binding arguments of a constant to an interpretation scope
    the scope is limited to the argument itself. It does not propagate to
    subterms but the subterms that, after interpretation of the notation,
    turn to be themselves arguments of a reference are interpreted
-   accordingly to the arguments scopes bound to this reference.
+   accordingly to the argument scopes bound to this reference.
 
    .. cmdv:: Arguments @qualid : clear scopes
 
-      Arguments scopes can be cleared with :n:`Arguments @qualid : clear scopes`.
+      Argument scopes can be cleared with :n:`Arguments @qualid : clear scopes`.
```

Comment:
It would make sense to reformulate differently: this command can be used to clear argument scopes of `qualid`.